src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -655,6 +655,11 @@ public ObservableCollection<string> AdditionalFiles
             }
         }
 
+        /// <summary>
+        /// Optional Markdown files list
+        /// </summary>
+        internal List<string> MarkdownFiles;
```

Comment:
seems this should be initialized to an empty list - though why is this a list and not an array, Ienumerable etc - I don't see it being used as a list (dynamic adding removing etc)

Reply:
Good point, Ienumerable or array should both work, will update